generator/templates/interface_header.tmpl

```diff
@@ -6,14 +6,23 @@
 
 #include "../<%= data.name.toLowerCase() %>.h"
 
+#include "../../generator/js_type_traits.h"
```

Comment:
These should be "non-relative" path. To resolve this issue, we can add the path to `include` object in index.js. Please see this line: https://github.com/romandev/node-addon-api/blob/master/test/binding.gyp#L26